### PR TITLE
Use separate trusted IP lists for ssh and VPN

### LINKED ISSUE #7: Use separate trusted IP lists for ssh and VPN
# 🚀 Feature Proposal

We should use separate trusted IP lists for ssh and VPN.

## Motivation

We need VPN to be accessible from any IP (since we have folks on travel) but we want to keep ssh restricted.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

In this PR we change the code to use separate trusted IP lists for ssh and VPN.  See also cisagov/openvpn-server-tf-module#14.

## 💭 Motivation and Context

We need VPN to be accessible from any IP (since we have folks on travel) but we want to keep ssh restricted.

## 🧪 Testing

All pre-commit tests pass.  This code has already been deployed to production and works great by all accounts.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: openvpn.tf [hunk 1]
```diff
@@ -27,7 +27,7 @@ locals {
 }
 
 module "openvpn" {
-  source = "github.com/cisagov/openvpn-server-tf-module"
+  source = "github.com/cisagov/openvpn-server-tf-module?ref=improvement%2Fseparate-trusted-ip-lists-for-ssh-and-vpn"
```
Comment (dav3r) [inline openvpn.tf]: Reminder: You'll want to remove this temporary testing tidbit...

**Review by dav3r — APPROVED** [review]
Looks good, except for one testing thang that needs to be excised. 🔪

File: openvpn.tf (hunk 1 above)
Comment (jsf9k) [reply openvpn.tf]: Yep, I forgot to remove that.  Glad you noticed it.  Removed in commit 0073279.

**Review by mcdonnnj — APPROVED** [review]
:shipit:

**Pull request merged**